src/java.base/share/classes/java/io/File.java

```diff
@@ -162,7 +165,8 @@
 "EnsuresNonNullIf(expression={\"list()\",\"list(FilenameFilter)\",\"listFiles()\",\"listFiles(FilenameFilter)\",\"listFiles(FileFilter)\"}, result=true)\""
 })
 @AnnotatedFor({"index", "initialization", "interning", "lock", "nullness"})
-public class File
+@DefaultQualifierForUse(Readonly.class)
+public @ReceiverDependentMutable class File
```

Comment:
Write the RDM consistently either on a line before the class declaration or like here. The files are mixed.

Reply:
Thanks!